K2Bridge.Tests.End2End/TestElasticClient.cs

```diff
@@ -139,47 +141,24 @@ public async Task<JToken> MSearch(string indexName, string jsonQueryFile, bool v
         }
 
         /// <summary>
-        /// API operation for wildcard index search (hitting the IndexList endpoint).
+        /// API operation for resolving indices (hitting the IndexList endpoint).
         /// </summary>
         /// <param name="optionalIndexToKeep">Optional input with index name to keep.
         /// if this is not null, all other index names will be removed and it will be
         /// normalized by removing the database name from kusto's db:table pair.</param>
         /// <returns><c>JToken</c> with parsed response.</returns>
         public async Task<JToken> Search(string optionalIndexToKeep = null)
```

Comment:
looks like its still optional (the index to keep). so why by default you pass the 'index' in the e2e test (commented there also)

Reply:
Yes, it should not be optional if we keep passing the index to ES also.

Reply:
Not needed anymore (removed the other changes).